src/filter/filter.tsx

```diff
@@ -153,11 +153,12 @@ const FilterBase = ({
                     {toggleFilterButtonLabel}
                 </FilterButton>
                 <Overlay show={visible} disableTransition>
-                    {visible ? (
-                        <FloatingWrapper>
+                    {
+                        <FloatingWrapper aria-hidden={!visible}>
```

Comment:
prefer use of `inert` which also prevents keyboard navigation via Tab

```suggestion
                        <FloatingWrapper inert={inertValue(!visible)}>
```